### DISCUSSION TITLE
Need Opinion and help on User Dahsboard

### DISCUSSION THREAD

**Discussion by Daniel-Alamezie** [swellstores/community#440] [Help] [answered]
Hi all,

So i have been working on market place using swell as the e-commerce backend, and atm i am working on the users dashboard. So atm all users can create accounts but also users that have accounts can choose to create a sellers account (can only be made by users with the platform account). Now the account can then be reviewed on  swell by admins for approval.

Now after sellers accounts have been approved, they can upload things for sale, now for transparency i need a way for the sellers to be notified when their items have been purchased.(p.s all items uploaded are digital). Admins on swell sell and fufil the sale, the sellers only get a cut for uploading their work on the platform for sale. 
Will i be able to achieve this using swell and if yes what approach would one follow.

Please dont hesitate to @ me and ask questions if anything is a bit unclear.

Thanks

**Comment by Daniel-Alamezie** [comment]
Or will this include me making a new custom model on the swell dashboard to make this happen?

**Comment by ericingram** [comment, marked as answer]
There are many ways to achieve this in Swell, so it would be hard to give you a solution in a short post. Generally I would recommend making new models for new concepts. Soon you will be able to manage records of custom models in the dashboard, but until then it feels like a limitation.

Email notifications can certainly be configured to do what you’re describing.
